about.html

```diff
@@ -13,59 +13,82 @@
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+        <header>
+            <div class="nav-container">
+                <div class="logo">
+                    <img src="img/lambda-black.png" alt="Logo">
+                </div>
+                <nav>
+                    <a href="index.html">Home</a>
+                    <a href="about.html">About</a>
+                    <a href="#">Products</a>
+                    <a href="#">Blog</a>
+                    <a href="contact.html">Contact</a>
+                </nav>
+            </div>
+            <div class="banner-container">
+                <img src="img/jumbo-about.png" alt="banner" >
+            </div>
+        </header>
+        <section class="container-1">
+          <h1>About</h1>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          </p>
+        </section>
+      
+        <section class="container-2">
+          <div class="wrapper-left">
+            <div class="img-container">
+                <img src="img/about-plan.png" alt="strategy">
+            </div>
+            <h2>Strategy</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            </p> <br>
```

Comment:
`<p>` are already a block, so you don't need a `<br>` tag to break the line, if you wanted more white space before the next block, it's better done with styles.